javascript/javascript-coding-conventions.md

```diff
@@ -217,3 +217,37 @@ for (var i = 0; i < counter; i++) {
 }
 console.timeEnd('array time'); // 40.505ms
 ```
+
+## 9. Return the same datatype in one function
```

Comment:
This example is not correct. Please use a function that returns both number and boolean as an example.